### PR TITLE
rewording words/sentences and adding small changes to the code

### PR DESCRIPTION
**Pull request by semacu**
These are some thoughts for changes following our recent [Data Carpentry in R in Cambridge 28th-28th June 2018](https://tavareshugo.github.io/2018-06-28-cambridge/). Please follow commit comments.

### CODE REVIEW

File: 04-visualization-ggplot2.Rmd [hunk 1]
```diff
@@ -529,6 +527,8 @@ ggplot(surveys_complete, aes(x = species_id, y = hindfoot_length)) +
 Faceting is a great tool for splitting one plot into multiple plots, but sometimes you may want to produce a single figure that contains multiple plots using different variables or even different data frames. The **`gridExtra`** package allows us to combine separate ggplots into a single figure using `grid.arrange()`:
 
 ```{r gridarrange-example, message=FALSE, purl=FALSE, fig.width=10}
+install.packages("gridExtra")
```
Comment (fmichonneau) [inline 04-visualization-ggplot2.Rmd]: can you please put this in its own chunk and using `eval=FALSE, purl=TRUE`:

```{r install-gridextra, eval=FALSE, purl=TRUE}
install.packages("gridExtra")
```

**Comment by fmichonneau** [comment]
Thanks for these edits. They look great. I just put a small technical comment about the `install.packages` that is best not being installed as we run the chunks (it's being installed when the lesson is being built so we don't need to do it twice).

**Comment by semacu** [comment]
Thanks!

**Comment by mondorescue** [comment]
@semacu your edits for 75b17e0 looks good. @fmichonneau, is @semacu's d9df873 commit good or does it needs correction regards to putting `install.packages("gridExtra")` in its own chunk?

**Comment by fmichonneau** [comment]
I updated the PR to put the installation of gridExtra in its own chunk. The rest of the PR looked great! Thanks for your work on this @semacu!

**Comment by semacu** [comment]
Thanks for developing and maintaning the course!
Sergio

**Pull request merged**